subxt/src/backend/rpc/reconnecting_rpc_client/mod.rs

```diff
@@ -0,0 +1,630 @@
+// Copyright 2019-2024 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! # reconnecting-jsonrpsee-ws-client
+//!
```

Comment:
Nit: once this is exposed it'd be good to fill this in with an example of constructing a subxt client using this reconnecting rpc client :)